web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -1134,3 +1134,29 @@ function joinup_core_post_update_delete_orphaned_solutions() {
     $solution->delete();
   }
 }
+
+/**
+ * Fix broken accounts with empty emails.
+ */
+function joinup_core_post_update_fix_broken_accounts(&$sandbox) {
```

Comment:
* This belongs more to `joinup_user`.
* Parameter `$sandbox` is not used.
* Missing return type.